dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -147,6 +149,9 @@ if (isMaster) {
         createNewWindow(url);
     });
     app.on('ready', () => {
+        // Remove the default electron menus, waiting for the application to set its own.
+        Menu.setApplicationMenu(null);
```

Comment:
https://github.com/theia-ide/theia/blob/021f69f8f4866ed2aee3f9ff750f59c38224eabe/packages/core/src/electron-browser/menu/electron-menu-contribution.ts#L81-L85 Does it work on Linux and Windows? I am checking it.

Reply:
Although we have platform-specific code for the menu contribution, it does not break anything on Windows.

I did not truly get what this PR is intended to fix besides the window title. Can you please try to explain it to me? (Or you can show it as a screencast if it is not so much extra work for you.) Thank you!